stubs/Exception.stub

```diff
@@ -52,6 +52,17 @@ interface Throwable
 
 class Exception implements Throwable
 {
+	/** @var string */
+	protected $message;
+
+	/** @var mixed */
```

Comment:
The only exception type I am aware of which doesnt has a `int` code is PDOException

Reply:
Yes me too.

But getCode is typed as returning mixed... so I think I have to match this

I'll wait for ondrej decision